MANIFEST.in

```diff
@@ -1 +0,0 @@
-include data/bash-completion/s4cmd
```

Comment:
this is only used with sdist command used for distribution should this be removed? https://docs.python.org/2/distutils/sourcedist.html#the-manifest-in-template

Reply:
Since the file is deleted, it is no longer needed. When it doesn't exist the default manifest is used: https://docs.python.org/2/distutils/sourcedist.html#manifest which works for s4cmd since it's just py files which are included by default